routes/logout.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const router = express.Router();
+
+router.get("/", async(req, res) => {
+    res.clearCookie("user");
+    res.redirect("/login");
+  });
```

Comment:
Run `npm run lint -- --fix`

Reply:
fixed in 1ce2e7aca0bbd21a4f8091041c32c6a311e98dda